### DISCUSSION TITLE
[FR] macOS M1/2 support + HomeBrew

### DISCUSSION THREAD

**Discussion by andrewcrook** [ivangabriele/clamav-desktop#68] [Feature Request Proposals]
- support Apple Silicon by adding a Apple Silicon dmg
- add casks to HomeBrew-casks or host a tap in GitHub

**Comment by ivangabriele** [comment]
@andrewcrook I don't know yet how I'll be able to build and test M1s (I'll surely have to ask for contributors help about that) since this is not yet supported by Github Actions VMs. It seems that [Github has it on its roadmap](https://github.com/actions/runner/issues/805#issuecomment-1438149537). But I'll definitely add that to the roadmap.

As Homebrew / tap I have no experience as a publisher but I'll check that and eventually add it to the roadmap. In any case it won't be for the v0.4.0. My first goal is to get a first fully-working version covering [these goals](https://github.com/ivangabriele/clamav-desktop#v040).
